sdk/core/azure-core-amqp/samples/internal/eventhub_get_properties_sample/eventhub_get_eventhub_properties_sample.cpp

```diff
@@ -75,7 +74,7 @@ EventHubProperties GetEventHubProperties(
       properties.PartitionIds.push_back(static_cast<std::string>(partition));
     }
   }
-  managementClient.Close();
+  managementClient.Close({});
```

Comment:
Could/should be written as `Close()`?

Reply:
Could, same answer as the previous 3 answers.